packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -213,10 +237,17 @@ async function ejectToBareAsync(projectRoot, options) {
   // for example RNGH and Reanimated. We should prefer the version that is already being used
   // in the project for those, but swap the react/react-native/react-native-unimodules versions
   // with the ones in the template.
-  let combinedDependencies = { ...defaultDependencies, ...pkgJson.dependencies };
-  combinedDependencies['react-native'] = defaultDependencies['react-native'];
-  combinedDependencies['react'] = defaultDependencies['react'];
-  combinedDependencies['react-native-unimodules'] = defaultDependencies['react-native-unimodules'];
+  const combinedDependencies = { ...defaultDependencies, ...pkgJson.dependencies };
+
+  for (const dependenciesKey of [
+    'react',
+    'react-dom',
+    'react-native-web',
+    'react-native-unimodules',
+    'react-native',
+  ]) {
+    combinedDependencies[dependenciesKey] = defaultDependencies[dependenciesKey];
```

Comment:
What happens if the `package.json` of the project has a dependency (e.g. `react-native-web`), but it doesn't exist in the template? It seems that it would become `"react-native-web": undefined` and be removed from dependencies.

Should we check that the key exists in `defaultDependencies` before copying it?